### DISCUSSION TITLE
Studio / UI does not take autogenerated UUID

### DISCUSSION THREAD

**Discussion by MentalGear** [supabase/supabase#9416] [General]
I set my column to default autogenerate UUID.

When I try to insert a row into the table via the studio(website), it asks for a manual uuid instead of generating one by default if left empty.

<img width="653" alt="Screenshot 2022-10-08 at 19 52 37" src="https://user-images.githubusercontent.com/2837147/194721171-5d4b1a74-c87b-4aa8-803e-937e291ef47f.png">

**Comment by GaryAustin1** [comment]
If you defined the column with generate_uuid_v4() as the default when you created the new column,  then when you insert and leave it blank it should (does for me) insert a UUID into the column automatically.
